### ISSUE TITLE
0.14.4 Setting resolution in game isn't working properly. 

### ISSUE THREAD

**Issue by DeathWrench** [Interkarma/daggerfall-unity#2434]
Using reshade I noticed that even if I render the game in 1080p but windows resolution is 4k the game just "stretches" the image. If you use RTGI shader than this is extremely performance intensive. 

setting
![image](https://user-images.githubusercontent.com/45341450/192805406-70047bbc-80f9-42da-98e6-a81d870c8e50.png)

Here is 1080p
![image](https://user-images.githubusercontent.com/45341450/192804639-c0262dda-69e8-4179-bebc-c30b93fe8a7b.png)
![image](https://user-images.githubusercontent.com/45341450/192805813-967d6736-fb2b-4a2e-a68b-99006fa5d135.png)

and 4k with resolution in game set to 1080p (notice the FPS counter drops substantially)
![image](https://user-images.githubusercontent.com/45341450/192805088-ab423b1a-ba67-4a5c-8541-27a2ddf477de.png)
![image](https://user-images.githubusercontent.com/45341450/192805671-963bdf9c-f857-466c-b762-ac408f91af04.png)

**Comment by Interkarma** [comment]
Hey there. :) Please use the forums or Discord for individual support. This issues tracker is for the base game only without mods.

https://forums.dfworkshop.net/

**Comment by DeathWrench** [comment]
This still is a base game issue, using ReShade is just how I was able to notice it. If it was working properly, ReShade would scale properly; it shouldn't be so shrunken. I'm assuming it's rendering the game in 4k still and then just downscaling the image? Any other game I set resolution to 1080p. ReShade scales appropriately, but as you can see in the above images, it doesn't. Currently the solution is to just set resolution to 1080p in windows, but kinda annoying to have to do this every time I launch the game.

**Comment by Interkarma** [comment]
I'm happy to work through the support process back and forth with you, but this is not the place for it. The forums are the correct place to ask questions and get help. If a bug is found, then the development team will raise with an issue to resolve.

https://forums.dfworkshop.net/

With that said, I'll try to explain what you're seeing. When it comes to fullscreen, there are two paths for this:

**Fullscreen Window** - The default setting. Game is rendered internally to your configured game resolution then scaled into a borderless window across the whole screen at your desktop resolution. Other windows (e.g. ReShade) will still appear at your desktop resolution, as this hasn't changed. The advantage of this mode is you can quickly alt+tab in and out of game, show desktop, etc. as the game is just another window. This mode is the most compatible and causes the fewest issues overall.

**Exclusive Fullscreen** - Can be enabled by setting "ExclusiveFullscreen=True" in settings.ini. This mode sets your display resolution to match game resolution and outputs to this exclusive view. The downside of exclusive fullscreen is a short delay when you alt+tab in and out of game, as the graphics card needs to switch back to desktop resolution and unload/reload textures from memory. This mode is fastest during play, but can have issues returning to game after alt+tab depending on system configuration, mods, drivers, etc. A lot more can go wrong, so it's not enabled by default.

What you're looking for is exclusive fullscreen. With this enabled, Unity will also set your video output to match the game resolution.

I hope the above proves helpful. If more support is needed, please use the forums link and open a topic in Help & Support. Thank you. :)

**Comment by DeathWrench** [comment]
I did try these commands before but didn't know about the settings.ini file. 
-screen-height 1080 -screen-width 1920 -window-mode exclusive

After fiddling with the settings.ini though it finally worked! Many thanks!

My apologies for being so persistent. I tried registering on the site, and I don't know if I entered my username or email wrong but I didn't get an activation email. Resending the activation email says that there is no account registered there, and trying to recover my account does the same thing. I couldn't find the discord server, just a fan server called Lysandus' Tomb. Is this the one? 
My discord is DeathWrench#6650 
I'm just gonna leave this here cause I don't want to bump this anymore lol.
